technologies/powr-echorand/overview.md

```diff
@@ -130,8 +130,8 @@ The following algorithm parameters are set by constants, or configured at the **
 | -----------: | :----------------------------------------------------- |
 | $Λ$          | "large" interval, the average time required to distribute a 1 MB message across the network |
 | $λ$          | "small" interval, the average time required to distribute a 256-bit message across the network |
-| $N_g$        | the number of block producers in a round, used in the function $VRF(r, 1)$ |
-| $N_c$        | the number of block verifiers in a round, used in the function $VRF(r, s), s > 1$ |
+| $N_g$        | the number of block producers in a round, used in the function $VRF(r, a, 1)$ |
+| $N_c$        | the number of block verifiers in a round, used in the function $VRF(r, a, s), s > 1$ |
 | $t_h$        | the threshold for making a positive decision when verifying, and can be selected by $0.69*N_{c}$ |
 | $μ$          | $4 + 3*k, k > 0$ - maximum number of algorithm steps after which an empty new block is created |
```

Comment:
here and elsewhere it does not take into account that empty blocks are not issued now